Dockerfile

```diff
@@ -83,7 +89,6 @@ RUN ["dash", "-c", "\
 "]
 
 # Configure Arduino CLI
-USER ${USER}
```

Comment:
This line was what was causing all the failed builds while I was iterating on this PR. See https://docs.github.com/en/actions/creating-actions/dockerfile-support-for-github-actions#user.

Reply:
https://github.com/actions/runner/issues/1876

Reply:
It's important to have this. Otherwise, `root` owns everything you build on your machine when you run the container locally. Then you don't have permissions to delete build artifacts, when you are outside the container.